DEVELOPER_GUIDE.md

```diff
@@ -131,6 +131,38 @@ curl localhost:9200
   "tagline" : "The OpenSearch Project: https://opensearch.org/"
 }
 ```
+
+Additionally, it is also possible to run a cluster with security enabled:
+```shell script
+./gradlew run -Dsecurity.enabled=true -Dhttps=true -Duser=admin -Dpassword=admin
```

Comment:
Looking at the build.gradle, `security.enabled=true` already sets the https, user, password system properties. Perhaps change this to just `./gradlew run -Dsecurity.enabled=true`